changelogs/fragments/vaultfilters.yml

```
@@ -0,0 +1,3 @@
+minor_changes:
+  - vault filter, create Ansible Vault to encrypt your data
+  - unvault filter, open an Ansible Vault to retrieve your data
```

This would be a great use for the changelog feature for announcing new filter/test plugins (https://docs.ansible.com/ansible/devel/community/development_process.html#changelog-fragment-entry-format-for-new-jinja2-plugins-roles-and-playbooks):
```suggestion
add plugin.filter:
  - name: vault
    description: Create Ansible Vault to encrypt your data
  - name: unvault
    description: Open an Ansible Vault to retrieve your data
```
That way, the filters will be announced as new filter plugins similar to other new plugins ([example](https://github.com/ansible-community/ansible-build-data/blob/main/4/CHANGELOG-v4.rst#filter-1)).